chalicelib/lib/reminder.py

```diff
@@ -0,0 +1,64 @@
+import json
+import logging
+from datetime import datetime
+from typing import Any, Dict, List, Optional
+
+import requests
+
+from chalicelib.lib.slack import Slack
```

Comment:
We already do this in the action baseclass. Can we provide this as an objet to this new function to remove duplication?

Reply:
Restructured a bit, not sure if it's better but at least we never create more than one Slack instance on real requests.